vllm/attention/selector.py

```diff
@@ -107,6 +107,11 @@ def _which_attn_to_use(
             "Cannot use FlashAttention-2 backend due to sliding window.")
         return _Backend.XFORMERS
 
+    if head_size % 32 != 0 or head_size > 256:
```

Comment:
https://github.com/vllm-project/vllm/blob/e941f885843d4bcd239f805a9267729e9631556f/vllm/attention/backends/flash_attn.py#L12

Can you add a comment this should be in sync with this. https://github.com/vllm-project/vllm/blob/e941f885843d4bcd239f805a9267729e9631556f/vllm/attention/backends/flash_attn.py#L12